setup.cfg

```diff
@@ -54,6 +54,8 @@ install_requires =
     inquirer~=3.0
     tabulate
     pygments~=2.0
+    # For automatic login
+    aiohttp~=3.8
```

Comment:
**non-blocking,question**: I like this approach of annotating a dependency with the reason why it was added. Do you think we should do this for all of our deps?

Reply:
We did it for a few of them, but the new CLI stuff was just "CLI utilities"